qds_sdk/cluster.py

```diff
@@ -1031,13 +1040,16 @@ def set_cluster_info(self, aws_access_key_id=None,
 
         `presto_custom_config`: Custom Presto configuration overrides.
 
+        `bastion_node_public_dns`: Public dns name of the bastian node. Required only if cluster is in private subnet.
+
         """
 
         self.disallow_cluster_termination = disallow_cluster_termination
         self.enable_ganglia_monitoring = enable_ganglia_monitoring
         self.node_bootstrap_file = node_bootstrap_file
         self.__set_node_configuration(master_instance_type, slave_instance_type, initial_nodes, max_nodes, slave_request_type, fallback_to_ondemand)
-        self.__set_ec2_settings(aws_access_key_id, aws_secret_access_key, aws_region, aws_availability_zone, vpc_id, subnet_id, role_instance_profile)
+        self.__set_ec2_settings(aws_access_key_id, aws_secret_access_key, aws_region, aws_availability_zone, vpc_id, subnet_id,
+                                bastion_node_public_dns, role_instance_profile)
```

Comment:
The order of arguments passed should be reversed. role_instance_profile followed by bastion_node_public_dns.